### ISSUE TITLE
"travis setup releases --com" is giving "Bad credentials" error

### ISSUE THREAD

**Issue by alexellis** [travis-ci/travis.rb#775]
I was able to run: `travis login --com --github-token $GH_PAT_TOKEN`

But I can't setup any releases. I've done this in the past, encrypting a token for publishing artifacts to GitHub Releases.

My GH username is alexellis, but I have 2FA enabled, and Travis is no longer asking me for that.

```
% travis setup releases --com

Username: alexellis
Password for alexellis: ***************************************************************************
Bad credentials
for a full error report, run travis report --pro
```

Travis' version is showing:

```
travis version
1.10.0
```

I've tried a few variations of versions after searching past issues.

This is for an [OSS project](https://github.com/alexellis/jaas) and will help us get automated builds in place

I'd appreciate any guidance you could give here.

**Comment by d-Rickyy-b** [comment]
Running into the same issue. Also having f2a enabled and trying to use a regular "personal access token" generated from https://github.com/settings/tokens.

Output of `travis report --pro`
```
System
Ruby:                     Ruby 2.5.5-p157
Operating System:         Ubuntu 18.04
RubyGems:                 RubyGems 2.7.6.2

CLI
Version:                  1.10.0
Plugins:                  none
Auto-Completion:          yes
Last Version Check:       2020-11-20 20:22:52 +0100

Session
API Endpoint:             https://api.travis-ci.com/
Logged In:                as "d-Rickyy-b"
Verify SSL:               yes
Enterprise:               no

Endpoints
org:                      https://api.travis-ci.org/ (access token)
com:                      https://api.travis-ci.com/ (access token, current)

Last Exception
An error occurred running `travis setup --pro`:
    RuntimeError: Not Found
        from /var/lib/gems/2.5.0/gems/travis-1.10.0/lib/travis/tools/github.rb:212:in `rescue in basic_auth'
        from /var/lib/gems/2.5.0/gems/travis-1.10.0/lib/travis/tools/github.rb:208:in `basic_auth'
        from /var/lib/gems/2.5.0/gems/travis-1.10.0/lib/travis/tools/github.rb:216:in `login'
        from /var/lib/gems/2.5.0/gems/travis-1.10.0/lib/travis/tools/github.rb:81:in `possible_tokens'
        from /var/lib/gems/2.5.0/gems/travis-1.10.0/lib/travis/tools/github.rb:50:in `each_token'
        from /var/lib/gems/2.5.0/gems/travis-1.10.0/lib/travis/tools/github.rb:37:in `with_token'
        from /var/lib/gems/2.5.0/gems/travis-1.10.0/lib/travis/cli/setup/releases.rb:12:in `block in run'
        from /var/lib/gems/2.5.0/gems/travis-1.10.0/lib/travis/cli/setup/service.rb:61:in `block in deploy'
        from /var/lib/gems/2.5.0/gems/travis-1.10.0/lib/travis/cli/setup/service.rb:52:in `configure'
        from /var/lib/gems/2.5.0/gems/travis-1.10.0/lib/travis/cli/setup/service.rb:60:in `deploy'
        from /var/lib/gems/2.5.0/gems/travis-1.10.0/lib/travis/cli/setup/releases.rb:11:in `run'
        from /var/lib/gems/2.5.0/gems/travis-1.10.0/lib/travis/cli/setup.rb:55:in `run'
        from /var/lib/gems/2.5.0/gems/travis-1.10.0/lib/travis/cli/command.rb:200:in `execute'
        from /var/lib/gems/2.5.0/gems/travis-1.10.0/lib/travis/cli.rb:64:in `run'
        from /var/lib/gems/2.5.0/gems/travis-1.10.0/bin/travis:18:in `<top (required)>'
        from /usr/local/bin/travis:23:in `load'
        from /usr/local/bin/travis:23:in `<main>'


For issues with the command line tool, please visit https://github.com/travis-ci/travis.rb/issues.
For Travis CI in general, go to https://github.com/travis-ci/travis-ci/issues or email support@travis-ci.com.
```

**Comment by d-Rickyy-b** [comment]
I created a [new personal access token](https://github.com/settings/tokens) for the Travis CLI and a new one for the releases integration. The CLI token needs `repo, user:email, read:org` permissions (check the [travis docs](https://docs.travis-ci.com/user/github-oauth-scopes/)). The releases token needs `repo.public_repo - Access public repositories` permissions only.

Sadly using the CLI to automate the whole process doesn't seem to work for whatever reason at the moment. So I worked around the issue by using the travis CLI like that:

```bash
$ travis login --pro --github-token <personal_access_token_CLI>
$ travis encrypt --pro <personal_access_token_RELEASES>
```

I entered the result of the second command in my travis.yml file under `deploy.api_key.secure`. This might not fix the issue but could provide a workaround.
